docs/dashboard/catalog/products.md

```diff
@@ -238,6 +242,22 @@ You can also add descriptions of specific product features to your images. Hover
 
 ![Editing product images](../screenshots/catalog-product-images.jpeg)
 
+### How to export products
+
+The CSV export tool allows you to export products from your dashboard to a CSV file. To export products go to _Catalog_ , then on the main _Products_ page click the three dots at the top of the page. You can export the whole product base or select and export specific products to a CSV file.
```

Comment:
I would add the one sentence that the CSV file will be sent via email.

Reply:
Okey, I'll add it.